pkg/actions/lua/storage/aws/glue.go

```diff
@@ -223,3 +225,37 @@ func getTable(c *GlueClient) lua.Function {
 		return 2
 	}
 }
+
+func createDatabase(c *GlueClient) lua.Function {
+	return func(l *lua.State) int {
+		client := c.client()
+		database := lua.CheckString(l, 1)
+		// AWS API call
+		_, err := client.CreateDatabase(c.ctx, &glue.CreateDatabaseInput{
+			DatabaseInput: &types.DatabaseInput{
+				Name:        aws.String(database),
+				Description: aws.String("Created by lakeFS Action"),
```

Comment:
This should probably be controlled by the user - and/or perhaps include the username who created the database.  As it is, it will probably be very hard to manage the created databases.